packages/core/src/mixins/source.js

```diff
@@ -199,7 +199,29 @@ module.exports = mixin(
             }
 
             _getTree() {
-                return fs.describe(this.fullPath, this.relPath, undefined, this.get('ext'));
+                const exclude = this.get('exclude');
+                const matchers = [];
+
+                function addMatcher(matcher) {
+                    if (_.isString(matcher)) {
+                        matchers.push(`${matcher}`);
+                    }
```

Comment:
This is weird, and I know it was like that before, but still 😅 We check if it’s a string but we always give it a string anyway, no? I guess we could remove this whole function and directly push on line 213 and 216, wdyt?

Reply:
hm, we could do that. i guess the check is here because the fractal config is not very type-safe, e.g. users can set almost any types of values for this option. but we don't really want to allow object etc to be added as matchers?

then again, anymatch allows functions, regex etc. so this check does not actually make any sense.

Reply:
Oh, looked at the previous code and now i get it - previously we needed to negate the matchers by prepending `!` so it had to be a string. but now we don't need the negation anymore, so we can allow non-string matchers as well.

actually we don't even need the array/push anymore, we can send it directly to anymatch :)

Reply:
Nice!